README.md

```diff
@@ -2,11 +2,66 @@
 
 # Introduction
 
-This image provides a reproducable way to install VOLTTRON within a docker container.
+This image provides a reproducible way to install VOLTTRON within a docker container.
 By using a volume mount of the `VOLTTRON_HOME` directory, runtime changes made by the platform are visible on the host and are preserved across instances of the container.
 Similarly, changes made from the host are reflect in the container.
 The image features gosu, which allows the non-root user executing the volttron platform inside the container to have the same UID as the host user running the container on the host system.
-In conjection with volume mounting of the directory, this ensures that file ownership and permissions in `VOLTTRON_HOME` match the host user, avoiding cases were root in the container leaves files inaccessible to a user without sudo permissions on the host.
+In conjunction with volume mounting of the directory, this ensures that file ownership and permissions in `VOLTTRON_HOME` match the host user, avoiding cases were root in the container leaves files inaccessible to a user without sudo permissions on the host.
+
+# Prerequisites
+
+* Docker
+* Docker-compose
+
+If you need to install docker and/or docker-compose, you can use the script in this repo. From the root level, execute the following command:
+
+```bash
+$ ./docker_install_ubuntu.sh
+```
+
+## Gotcha
```

Comment:
This is not a Gotcha....it is a just a way of making it repeatable at a set point in time.  This is not necessary unless you want to build it locally.  Which I believe is why I didn't have the build in the docker-compose file originally.

Reply:
I understand now. Will rename and move to different section.